project/npda/migrations/0001_initial.py

```diff
@@ -32,7 +33,13 @@ class Migration(migrations.Migration):
                 ),
                 (
                     "nhs_number",
-                    models.CharField(max_length=10, verbose_name="NHS Number"),
+                    models.CharField(
```

Comment:
Can this change be done in an additional migration? Every time we edit an existing one now we need to go in and drop the staging database

Reply:
Spoke IRL - I will change this to be an additional migration